builtin/providers/rancher/provider.go

```diff
@@ -71,7 +73,7 @@ func init() {
 
 		"secret_key": "API secret used to authenticate with the rancher server",
 
-		"api_url": "The URL to the rancher API",
+		"api_url": "The URL to the rancher API, must include version uri (ie. v1 or v2-beta)",
```

Comment:
I suppose the new code won't work with v1 though, and the CATTLE_* variables still return the v1 URL.